src/components/performance/views.tsx

```diff
@@ -0,0 +1,200 @@
+  readonly downloadLink: string;
+  readonly metric: string;
+  readonly units: string;
+  readonly format?: string;
+  readonly title: ReactNode;
+  readonly titleText?: string;
+  readonly description: ReactNode;
+  readonly chart: SVGElement;
+  readonly children: ReactNode;
+}
+
+interface IMetricPageProperties {
+  readonly applicationCount?: ReadonlyArray<IMetricSerie>;
+  readonly linkTo: RouteLinker;
+  readonly organizations?: ReadonlyArray<IMetricSerie>;
+  readonly period: moment.Duration;
+  readonly region: string;
+  readonly serviceCount?: ReadonlyArray<IMetricSerie>;
+}
+
+export function latestValue(data: IMetricSerie): number {
+  return data.metrics.reduce((value, m) => (!isNaN(m.value) ? m.value : value), 0);
+}
+
+function Metric(props: IMetricProperties): ReactElement {
+  const downloadLink = parseURL(props.downloadLink, {
+    metric: props.metric,
+    units: props.units,
+  });
+
+  return (
+    <div className="govuk-grid-row govuk-!-padding-bottom-9">
+      <div className="govuk-grid-column-full">
+        <h2 className="govuk-heading-m" id={props.id}>
+          {props.title}
+        </h2>
+      </div>
+
+      <div className="govuk-grid-column-full">
+        <p className="govuk-body">{props.description}</p>
+
+        <MetricChart chart={props.chart.outerHTML} />
+
+        {props.children}
+
+        <div className="govuk-grid-column-full">
+          <a href={downloadLink.toString()} className="govuk-link" download>
+            Download &quot;{props.titleText || props.title}&quot; as a CSV
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export function MetricPage(props: IMetricPageProperties): ReactElement {
+  return (
+    <div className="govuk-width-container">
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds-from-desktop">
+          <h1 className="govuk-heading-xl">
+            <span className="govuk-caption-xl">GOV.UK Platform as a Service</span> {}
+            Performance dashboard
+          </h1>
+          <p className="govuk-body">
+            GOV.UK PaaS is a shared platform that public sector service teams can use to quickly host their applications
+            in the cloud.
+          </p>
+
+          <p className="govuk-body">
+            Each point on the charts is aggregated data over {}
+            <strong className="non-breaking">{props.period.humanize()}</strong>.
+          </p>
+
+          <hr className="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+
+          <div className="govuk-grid-row govuk-!-padding-bottom-9">
+            <div className="govuk-grid-column-full">
+              <h2 className="govuk-heading-m" id="uptime">
+                Uptime
+              </h2>
+            </div>
+
+            <div className="govuk-grid-column-full">
+              <p className="govuk-body">
+                Uptime is measured as the availability of the GOV.UK PaaS API on the platform.
```

Comment:
Uptime is measured as the availability of the GOV.UK PaaS API.

Remove "on the platform"